OpenRA.Mods.Common/Traits/Multipliers/HarvesterResourceMultiplier.cs

```diff
@@ -0,0 +1,33 @@
+﻿#region Copyright & License Information
+/*
+ * Copyright 2007-2018 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Traits
+{
+	[Desc("Modifies the bale values of this harvester.")]
+	public class HarvesterResourceMultiplierInfo : ConditionalTraitInfo
+	{
+		[FieldLoader.Require]
+		[Desc("Percentage modifier to apply.")]
+		public readonly int Modifier = 100;
```

Comment:
This field does not need a default value because of `FieldLoader.Require`.

Reply:
All our other *Multiplier traits define a default of 100 even tho they have `FieldLoader.Require`.

Reply:
Ok, then let's keep this. (Unless @pchote and @reaperrr agree that we should change the "convention" here.)

Reply:
By the way: `ExternalResourceMultiplier` does not have `100` as default for `Modifier`.

Reply:
`ExternalResourceMultiplier` is a bit of a different story. Seems like Graion didn't even put it in Multipliers folder which is understanable. It isn't something that modifies a value of its own traits. A `Modifier` of 125 doesn't increase your income by 25% in ExternalResourceMultiplier.

Would be a good idea if i could get your opinion on if we should put it on Multipliers folder or not too.

Regardless having the default for consistancy could be better tho.

Reply:
Nah, axe the defaults, I messed this one up.